### PR TITLE
weekly release notes update for Studio

### PR DESCRIPTION
**Pull request by DanielDamMx**

### CODE REVIEW

File: content/releasenotes/studio/7.23-and-above.md [hunk 1]
```diff
@@ -25,6 +25,20 @@ For updates on the operational status of Studio as well as deployment features a
 
 ## 2019
 
+### July 19th, 2019 ###
+
+#### Improvements
+
+* In the page editor, the **Layout Grid** widget has now been replaced by multiple widgets each representing a layout grid configured with a specific number of columns. This improvement is based on user feedback and makes building (responsive) pages a lot easier. Find the new pre-configured widgets in the **Columns** category of the widgets toolbox in the page editor.
```
Comment (MarkvanMents) [inline content/releasenotes/studio/7.23-and-above.md]: Is there a specific piece of feedback we could point to?

File: content/releasenotes/studio/7.23-and-above.md [hunk 2]
```diff
@@ -25,6 +25,20 @@ For updates on the operational status of Studio as well as deployment features a
...
+* You can now split layout grid rows into six columns with the click of a button.
+* For **App from a spreadsheet**, you can now import spreadsheets up to a maximum size of 1 MB.
```
Comment (MarkvanMents) [inline content/releasenotes/studio/7.23-and-above.md]: Do we need *maximum size of*, could just say `spreadsheets up to 1MB`

**Comment by DanielDamMx** [comment]
Hi Mark, there isn’t. It’s general user observations.
I’m on holiday from now on – please direct your questions to Arjan H !

See you soon.

**Comment by DanielDamMx** [comment]
Fine by me!

**Pull request merged**